run-tests.bash

```diff
@@ -21,5 +21,6 @@ cargo test --no-default-features --features compress-zlib-ng
 cargo test --no-default-features --features form
 cargo test --no-default-features --features multipart-form
 cargo test --no-default-features --features json
-cargo test --no-default-features --features tls
-cargo test --no-default-features --features tls-rustls
+cargo test --no-default-features --features tls-native
```

Comment:
Can you also test `tls-native-vendored`? Just so we know it builds properly

Reply:
Good call, updated!